service/http.directory/src/main/java/org/kaazing/gateway/service/http/directory/HttpDirectoryServiceHandler.java

```diff
@@ -148,9 +149,19 @@ protected void doSessionOpened(HttpAcceptSession session) throws Exception {
         }
 
         if (!underBaseDir) {
-            reportError(session, HttpStatus.CLIENT_BAD_REQUEST);
-            session.close(false);
-            return;
+        	boolean symbolicChecker = Files.isSymbolicLink(requestFile.toPath());
+        	if(symbolicChecker)
+        	{
+        		reportError(session, HttpStatus.CLIENT_NOT_FOUND);
```

Comment:
The formatting here looks funky, can we use the kaazing formatter @ https://github.com/kaazing/code.quality/blob/develop/src/main/resources/org/kaazing/code/quality/eclipse/CodeFormatterProfile.xml